examples/theano/robo_cartpole.py

```diff
@@ -0,0 +1,42 @@
+import gym, roboschool
+import numpy as np
+import random
+
+class NaivePolicy:
+    "Simple multi-layer perceptron policy, no internal state"
+    def __init__(self, observation_space, action_space):
+        return
+
+    def act(self, ob):
+        x = np.array([0.5-random.random()])
+        return x
+
+def demo_run():
+    env = gym.make("RoboschoolInvertedPendulum-v1")
+    pi = NaivePolicy(env.observation_space, env.action_space)
+
+    while 1:
+        frame = 0
+        score = 0
+        restart_delay = 0
+        obs = env.reset()
+
+        while 1:
+            a = pi.act(obs)
+            obs, r, done, _ = env.step(a)
+            score += r
+            frame += 1
+            still_open = env.render("human")
+            if still_open==False:
```

Comment:
What is the purpose of this guard? To safely close the environment? You can do so by calling env.close()

Reply:
without this, the program will not exit if the render window is closed by clicking GUI close button.

Reply:
If roboshool has odd rendering behavior which needs to be managed, it would be best to handle it with a custom Viewer class.